Testscripts/Windows/PERF-LKP.ps1

```diff
@@ -0,0 +1,248 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+param(
+    [object] $AllVmData,
+    [object] $CurrentTestData,
+    [object] $TestProvider,
+    [object] $TestParams
+)
+
+$TEST_SCRIPT = "perf_lkp.sh"
+$LKP_RESULTS = "lkp-results.log"
+$LKP_OUPUT = "lkp-output.log"
+$LKP_LOG_FILE = "lkp_log.tar"
+$LKP_LOG_DIR = "lkp_log"
+$VERTICAL_FORMAT_TESTS = @("oltp", "redis", "perf-bench-numa-mem", "apachebench", "pmbench", "aim9")
+$MICROBENCHMARK_TEST_NAMES = @("perf-bench-sched-pipe", "schbench", "oltp", "redis", "perf-bench-numa-mem", "apachebench", "pmbench")
+
+function Get-BenchmarkResult ($testName, $jsonResult) {
+    switch ($testName) {
+        "hackbench" {
+            $resultTitle = "{0,-30} {1,-30}" -f "throughput", "workload"
```

Comment:
Using '30' & '-30' in many places. Where are they coming from?

Reply:
It's just a format width. I estimated.

Reply:
Hope you don't fix those numbers again.